src/main/java/org/generationcp/middleware/service/api/study/StudyEntryDto.java

```diff
@@ -0,0 +1,119 @@
+package org.generationcp.middleware.service.api.study;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import org.pojomatic.Pojomatic;
+import org.pojomatic.annotations.AutoProperty;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@AutoProperty
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class StudyEntryDto {
```

Comment:
@abatac , as discussed, let's see if we can eventually adjust code and phase out StudyGermplasmDto. It would be good to maintain only one of the 2

Reply:
agree, we should start using this one